packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -400,7 +400,13 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
     const initialsName = !localParticipant.displayName ? '' : localParticipant.displayName;
 
     return (
-      <Stack key="local-video-tile-key" tabIndex={0} aria-label={strings.localVideoMovementLabel} role={'dialog'}>
+      <Stack
+        style={{ width: '100%', height: '100%' }}
```

Comment:
Should we define these styles in a constant or css file?

Reply:
yup I can find a place for these!